src/backup_domain.rs

```diff
@@ -22,3 +22,53 @@ use crate::pac::BKP;
 pub struct BackupDomain {
     pub(crate) _regs: BKP,
 }
+
+macro_rules! write_drx {
+    ($self:ident, $drx:ident, $idx:expr, $new:expr) => {
+        unsafe {
```

Comment:
with 0.8 you don't need unsafe here
https://docs.rs/stm32f1/0.8.0/stm32f1/stm32f103/bkp/dr/struct._DW.html

Reply:
Fixed, thanks!